src/resolve/resolved-link.h

```diff
@@ -26,6 +26,7 @@ struct LinkAddress {
 
         int family;
         union in_addr_union in_addr;
+        union in_addr_union in_brd;
```

Comment:
let's not abbreviate needlessly. "in_addr" is a abbreviation that is pretty common in libc, there's after all "struct in_addr". But "in_brd" is just cryptic. it's fine being terse for local variables, but this is a non-statically used structure. Hence let's maybe not be quite as cryptic. i.e. "in_addr_broadcast" or so?

Reply:
Sounds good.